### PR TITLE
Include Manual & Datasheet

### PR DESCRIPTION
**Pull request by dimitrisnl**
edit: issues resolved with upgrading to landr@2.2.0

### CODE REVIEW

**Comment by dimitrisnl** [comment]
Quite a lot of diffs (deleted files), maybe I need to include the build output?

**Comment by thgreasi** [comment]
Feels weird to me that we have a so large -diff, let's hear what the rest think.
Would it work if we placed the pdfs in the assets folder, right next to the favicon?

**Comment by thgreasi** [comment]
I don't think that it's a good idea to include the pdf's in `an object with all items from /assets`

**Comment by LucianBuzzo** [comment]
@dimitrisnl Can you please remove the `.DS_store` files and exclude them in `.gitignore` ?

**Comment by dimitrisnl** [comment]
What about the number of diffs? @LucianBuzzo

File: www/dist/react-static-routes.js [hunk 1]
```diff
@@ -2,11 +2,11 @@
     import React, { Component } from 'react'
     import { Route } from 'react-router-dom'
 
-    import _Users_lucianbuzzo_projects_fin_site_theme_landr_theme_fin_pages_Home from '/Users/lucianbuzzo/projects/fin-site/theme/landr-theme-fin/pages/Home'
-import _Users_lucianbuzzo_projects_fin_site_theme_landr_theme_fin_pages____ from '/Users/lucianbuzzo/projects/fin-site/theme/landr-theme-fin/pages/404'
+    import _Users_dnlytras_Documents_Projects_fin_site_theme_landr_theme_fin_pages_Home from '/Users/dnlytras/Documents/Projects/fin-site/theme/landr-theme-fin/pages/Home'
+import _Users_dnlytras_Documents_Projects_fin_site_theme_landr_theme_fin_pages____ from '/Users/dnlytras/Documents/Projects/fin-site/theme/landr-theme-fin/pages/404'
```
Comment (dimitrisnl) [inline www/dist/react-static-routes.js]: This isn't really nice to have.

Comment (LucianBuzzo) [reply www/dist/react-static-routes.js]: @dimitrisnl can we gitignore the dist directory? It seems that the balena site doesn't include the dist directory https://github.com/resin-os/balena

Comment (LucianBuzzo) [reply www/dist/react-static-routes.js]: We should create an issue on landr to remove these user specific paths as well

Comment (dimitrisnl) [reply www/dist/react-static-routes.js]: @LucianBuzzo Removed. Assuming this gets merged, do I just prepare a build locally and run the deploy script?

Does it get deployed to GH pages?

**Comment by dimitrisnl** [comment]
@LucianBuzzo I'm not sure what's the deal with this issue. It's not a direct dependency of ours.
Did you try to return `npm ci`?

**Comment by dimitrisnl** [comment]
@thgreasi Can you clone this for a moment? See if you have any issues with running it?

**Pull request merged**